src/components/EntityLookup.vue

```diff
@@ -78,6 +78,9 @@ export default Vue.extend( {
 		},
 	},
 	watch: {
+		disabled( isDisabled: boolean ): void {
+			this.search = isDisabled ? '' : this.search;
```

Comment:
I'm not sure what were gaining by assigning `this.search` to itself.

What do you think about the following?

```suggestion
			if ( isDisabled ) {
			  this.search = '';
			}
```

Reply:
much cleaner